### PR TITLE
Add config examples for inner ring application

### LINKED ISSUE #1358: Provide Inner Ring config example
It'd be nice to have config example for `neofs-ir` application similar to `neofs-node`. See `config/example`.

### PR DESCRIPTION
**Pull request by alexvanin**
Closes #1358

### CODE REVIEW

File: config/example/ir.env [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+NEOFS_IR_WITHOUT_MAINNET=false
+
+# Timeout for RPC client connection to sidechain
+NEOFS_IR_MORPH_DIAL_TIMEOUT=5s
+# List of websocket endpoints in sidechain RPC nodes
+NEOFS_IR_MORPH_ENDPOINT_CLIENT="wss://sidechain1.fs.neo.org:30333/ws wss://sidechain2.fs.neo.org:30333/ws"
+# List of hex-encoded 33-byte public keys of sidechain validators to vote for at application startup
+NEOFS_IR_MORPH_VALIDATORS="0283120f4c8c1fc1d792af5063d2def9da5fddc90bc1384de7fcfdda33c3860170"
+
+# Timeout for RPC client connection; ignore if mainchain is disabled
```
[the remark this answers is not in the document]
Comment (alexvanin) [reply config/example/ir.env:21]: I address reader there.

**Comment by alexvanin** [comment]
> Do we need json example?

I hope no :smiley:

**Review by carpawell — APPROVED** [review]

File: config/example/ir.env [hunk 2]
```diff
@@ -0,0 +1,165 @@
+# Logger level: one of "debug", "info" (default), "warn", "error", "dpanic", "panic", "fatal"
```
Comment (cthulhu-rider) [inline config/example/ir.env:1]: For storage node, we've decided to provide docs in single place, e.g. yaml. I suggest to do it for IR as well.

**Review by cthulhu-rider — APPROVED** [review]

**Comment by cthulhu-rider** [comment]
@alexvanin pls mention new files in readme.

File: config/example/ir.env (hunk 2 above)
Comment (alexvanin) [reply config/example/ir.env]: Fixed https://github.com/nspcc-dev/neofs-node/pull/1364/commits/4ac0dccf081df1c25885e5c81c7419ed18a54984

**Pull request merged**